### DISCUSSION TITLE
Install Difficulty - v5.1.1

### DISCUSSION THREAD

**Discussion by wmcgrathled** [openbroadcaster/observer#64] [Q&A]
I am hoping someone can help me out.  I am successfully running pyrate and am now pretty good at running the software.  I am trying to install 5.1.1 server on a fresh Raspberry PI (Bullseye).  I wrote the following steps that I performed for my reference but also to help with documentation on installation for less skilled people.  

The issue is that when I upload media (.mp3 for example) after uploading it gets stuck at:
**Processing Media...**

Here's the steps I performed:

`
Open Broadcaster Server setup:

1. Install Lite 64bit version of Raspi OS (Bullseye)
2. sudo apt update and upgrade
3. Install Git:
	sudo apt install git
4. Create user obsuser:
	sudo adduser obsuser
5. add to sudoers with:
	sudo usermod obsuser -a -G pi,adm,dialout,cdrom,sudo,audio,video,plugdev,games,users,input,netdev,spi,i2c,gpio	
6. Change to user obsuser, cd, mkdir gits, cd gits
7. sudo git clone https://github.com/openbroadcaster/observer.git
8. cd observer, sudo bash ob.installer.sh  (takes a while)
9. When finished record the last screen with the passwords, etc.
10. For good measure, sudo apt update / upgrade /autoremove
11. Since bullseye does not have avconv, we need to symbolic link to ffmpeg:
	ln -s /usr/bin/ffmpeg /usr/local/bin/avconv
	ln -s /usr/bin/ffprobe /usr/local/bin/avprobe
12. Navigate to http://(SERVER_IP)/updates/   and login with: updates/updates
13. The "Install Checker" should be all green.
14. Click "run updates now" and let it finish
15. Go to http://(SERVER_IP)   and login with: admin/[password from the file in step 9], the console should come up.
16. Go to media, new and see if you can upload a song.
`
I cannot figure out why it gets stuck there.

I should also mention I am "all green" on the "OpenBroadcaster Install Checker" and have run the updates.

**Comment by brookgagnon** [comment]
Could you share a screenshot where it gets stuck? Thanks!

**Reply by wmcgrathled** [reply]
Sure:
After dropping the .mp3 file in media add (working):
![image](https://user-images.githubusercontent.com/105522544/172428156-c5775b3e-165a-44d2-8384-7c02e361810d.png)

After it hits 100% of uploading (it will stick here forever):
![image](https://user-images.githubusercontent.com/105522544/172428538-461218af-5794-45ea-8623-9ecc84af3093.png)

**Reply by brookgagnon** [reply]
Are you able to check the PHP error log? That's most likely where an indication of something wrong would be (permissions error, etc.). We're working on better debug logging but until then it's a bit more of a manual process.

If you let me know the OS and install method for OB I might be able to point you in the right direction for error log, if you're not sure.

**Reply by mamimsllc** [reply]
I am having that same problem as well and it see this in the logs:

2022/06/13 14:52:53 [error] 24941#24941: *25 client intended to send too large body: 78910739 bytes, client: 173.50.79.57, server: _, request: "POST /upload.php HTTP/1.1", host: "xx.xx.xx", referrer: "http://xx.xx.xx"
2022/06/13 15:27:45 [error] 446#446: *7 client intended to send too large body: 78910739 bytes, client: 173.50.79.57, server: _, request: "POST /upload.php HTTP/1.1", host: "xx.xx.xx", referrer: "http://xx.xx.xx/"
2022/06/13 15:32:23 [error] 446#446: *36 client intended to send too large body: 78910739 bytes, client: 173.50.79.57, server: _, request: "POST /upload.php HTTP/1.1", host: "xx.xx.xx", referrer: "http://xx.xx.xx/"

**Reply by mamimsllc** [reply]
I added the following to the Nginx.conf file:

client_max_body_size 100m and after it tried to process the file but I received This File format is not supported.  This was further along than where I was before

**Reply by brookgagnon** [reply]
Do you have all the file formats enabled in admin -> media settings -> formats?

![image](https://user-images.githubusercontent.com/709130/173445949-a3753848-da89-4450-9eaa-008b86a8338c.png)

Sometimes the extension doesn't match the actual detected format.

**Reply by mimsma** [reply]
Works like a champ
